### PR TITLE
Update setup-glassfish.py to work with Payara 5

### PR DESCRIPTION
**Pull request by ajkyffin**
see https://github.com/icatproject/icatproject.github.io/pull/16

### CODE REVIEW

**Comment by MRichards99** [comment]
Tested the script and works with Payara 5.2021.1 - used ICAT Ansible and run on DataGateway API's GitHub Actions ([logs for this run](https://pipelines.actions.githubusercontent.com/ZF71EcVgERFtIDbsQ006XqQsiTy8DTnXrNQtfsech7QHMM7f5v/_apis/pipelines/1/runs/134/signedlogcontent/11?urlExpires=2021-03-04T10%3A38%3A48.7149232Z&urlSigningMethod=HMACV1&urlSignature=pSbn8nVxbXCX36YDRDLHj2UL2HmYlMKaYVD%2B9AqV2is%3D))

File: src/main/scripts/setup-glassfish.py [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 
 import threading
 import platform
 import shlex
 import subprocess
-try:
-    import io
-except ImportError:
-    import StringIO as io
+import io
```
Comment (MRichards99) [inline src/main/scripts/setup-glassfish.py:9]: Should this be reverted back to the try except statement since I guess that made this import work on 2 & 3?

Comment (ajkyffin) [reply src/main/scripts/setup-glassfish.py:9]: The `io` module is in Python 2.6+. I've tested this with Python 2.7 and 3.6.

Comment (MRichards99) [reply src/main/scripts/setup-glassfish.py:9]: Ah fair enough, just thought I'd check!

**Review by MRichards99 — APPROVED** [review]

**Pull request merged**